packages/console/src/pages/Users/index.tsx

```diff
@@ -65,9 +66,17 @@ const Users = () => {
             </tr>
           )}
           {isLoading && (
-            <tr>
-              <td colSpan={2}>loading</td>
-            </tr>
+            <TableLoading>
+              <td className={styles.userName}>
+                <ItemPreviewLoading />
+              </td>
+              <td>
+                <div />
+              </td>
+              <td>
+                <div />
+              </td>
+            </TableLoading>
```

Comment:
the usage of this `TableLoading` seems verbose, any good idea? @logto-io/eng

Reply:
The main reason is that every table may have different number of columns, and each column's loading block may be different as well.

Reply:
Also, column width is customized by classNames.

Reply:
I think we should not control the column width using css class names on each column. Should use `<col />` instead, IMO. Then we can avoid defining the <td>s in the loading skeleton.